### ISSUE TITLE
[Bug] Vim mode in AnuPpuccin does not work well.

### ISSUE THREAD

**Issue by PowapowaP** [AnubisNekhet/AnuPpuccin#180]
**Describe the bug**
When the cursor is on the heading and on the second character or later, if you press the “j”, it will jump to the last character, and then continue to press "j", there is no way to move. 
Simply , sometimes "j" and "k" does not work well in vim mode.

![image](https://github.com/AnubisNekhet/AnuPpuccin/assets/36119074/3a89be8b-5707-4a6f-b1dc-fb33b18a21e5)

**Comment by AnubisNekhet** [comment]
I think this is an Obsidian issue rather than a theme issue. Can you replicate this in the default theme?

**Comment by PowapowaP** [comment]
> I think this is an Obsidian issue rather than a theme issue. Can you replicate this in the default theme?

The default theme and the other theme (include Blue Topaz,Cybertron,Minimal,80s Neon) don`t replicate this.

But just now I disabled Style Settings, vim mode worked well.
So I think this is something wrong in Style Settings.

![image](https://github.com/AnubisNekhet/AnuPpuccin/assets/36119074/6ea270f4-9f25-4a86-a6a2-bfbb06b24495)
And I disabled the Custom Heading Margin, vim mode works fine now.
